packages/usa-in-page-navigation/src/styles/_usa-in-page-navigation.scss

```diff
@@ -82,12 +95,12 @@
 
     .usa-in-page-nav__item {
       @include u-border(0);
+      font-size: font-size($theme-in-page-nav-font-family, "2xs");
+      font-weight: bold;
       position: relative;
 
       &.usa-in-page-nav__item--sub-item {
-        a {
```

Comment:
@thisisdano we lost the second level indent here. Is that preferred? I thought it looked weird with just the top level being bold. Here are screen shots comparing both. Before pushing this back, I wanted to confirm with you, should we leave the indent out? Thanks

<img width="235" alt="image" src="https://user-images.githubusercontent.com/19338309/200526022-9c010ddc-6f19-41ca-9bd9-fcf6a2811ee3.png">


<img width="266" alt="image" src="https://user-images.githubusercontent.com/19338309/200525822-f6e6eb55-2945-4227-bbe9-6d2a36b61802.png">

Reply:
Yes, we're leaving the indent out to simplify presentation since this component is restricted to only two levels